docs/oid-info.md

```diff
@@ -177,6 +178,10 @@ Event or action that initiated the build. For example: `push`.
 
 Run Invocation URL to uniquely identify the build execution. SHOULD be fully qualified. For example: `https://github.com/example/repository/actions/runs/1536140711/attempts/1`.
 
+### 1.3.6.1.4.1.57264.1.22 | Source Repository Visibility At Signing
+
+Source repository visibility at the time of signing the certificate. MAY be empty if there is no Source Repository Visibility information available. For example: `private` or `public`.
+
```

Comment:
@feelepxyz can we clarify what we mean by "repository" in this context? GitLab uses the term "project" to refer to the namespace that encapsulates a git repository (and other features). We support visibility/permission toggles on a per-feature basis, so it is possible for a project to be public while the repository feature itself is private/disabled.

What if `Run Invocation URI` is not publicly accessible, for example? Can `Source Repository Visibility At Signing` be `public` despite `Build Config URI` pointing to a private project?

Should the intent be for this extension to broadly describe the visibility of all relevant features?

Reply:
@haydentherapper sorry for the late feedback. I was typing out as you merged.

Reply:
> it is possible for a project to be public while the repository feature itself is private/disabled.

@marshall007 ah right, in this case would it mean that the `Source Repository URI` would not be reachable without authentication? 

> Should the intent be for this extension to broadly describe the visibility of all relevant features?

I would imagine the extension would broadly describe the publicly accessible source URIs, wether that is the run invocation uri or source repository uri. 

Is `project_visibility` a good proxy for this?

Reply:
> I would imagine the extension would broadly describe the publicly accessible source URIs, wether that is the run invocation uri or source repository uri.

@feelepxyz yes, I agree and I think `project_visibility` is sufficient in signalling the intent.